e2e_samples/dataset_versioning/docs/adrs/source_data_store.md

```diff
@@ -0,0 +1,19 @@
+# Source Data Store
+
+## Status
+
+Decided
+
+## Context
+
+In order to demonstrate data versioning, we need to pull from a data store. There is no existing _source_ data store, so we need to decide which technology we would like to use.
+
+## Decision
+
+We have decided on `Azure SQL` as our data store.
+
+## Consequences
+
+* `Azure SQL` provides an inexpensive solution
+* Reduces the amount of unfamiliar technology introduced into this sample
```

Comment:
nit: In addition to it, we may be able to say "SQL is popular solution for data engineer, so it can make them to catch up easily".

This is nit, because what this sentence describe is obvious in engineering world.

Reply:
OK I added another bullet point